api/python/provisioner/config.py

```diff
@@ -542,3 +542,11 @@ class CortxResourceT(Enum):
     """Resource types in CORTX provisioner"""
 
     REPOS = "cortx_repos"
+
+
+class HashType(Enum):
+
+    """Hash type enumeration."""
+
+    MD5 = "md5"
+    SHA256 = "sha256"
```

Comment:
Should we consider SHA512 as well here? (It's quite popular and preferred over SHA256)

Reply:
Yes, I added that